apple/Handlers/RNNativeViewHandler.mm

```diff
@@ -40,6 +42,14 @@ - (void)touchesBegan:(NSSet<RNGHUITouch *> *)touches withEvent:(UIEvent *)event
 {
   [_gestureHandler setCurrentPointerType:event];
   [_gestureHandler.pointerTracker touchesBegan:touches withEvent:event];
+
+#ifdef RCT_NEW_ARCH_ENABLED
+  if ([self.view.superview isKindOfClass:[RCTParagraphComponentView class]]) {
```

Comment:
What's the structure on the new arch? Wouldn't the gesture be attached to the paragraph component view?

Reply:
The gesture itself is attached to `RCTParagraphView`. Unfortunately this class is not exported - that's why I've decided to check for superview (i.e. `RCTParagraphComponentView`)